README.md

```diff
@@ -11,6 +11,7 @@ A melting pot for all kinds of extensions to
 
 Check out [junit-pioneer.org](http://junit-pioneer.org), particularly [the documentation section](http://junit-pioneer.org/docs).
 
+
```

Comment:
Nitpick: Is there a reason why some sections are separated with two linebreaks (like here) and others with one (like "Contributing")?

Reply:
There's always a reason, @beatngu13. :grin: To better navigate a file, I put two empty lines before each top-level section - in most documents, that's the level 2 headers.